libdispatch/nch5s3comms.c

```diff
@@ -92,7 +92,7 @@
 
 #include "netcdf.h"
 #include "ncuri.h"
-#include "ncutil.h"
+#include "netcdf_vutils.h"
```

Comment:
This seems to cause by default builds config to fail! Since there's not such header.